js/lykke2.js

```diff
@@ -482,7 +484,7 @@ module.exports = class lykke2 extends Exchange {
         return this.parseTickers (tickers, symbols);
     }
 
-    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+    async fetchOrderBook (symbol, limit = 0, params = {}) {
```

Comment:
the signature here is wrong, it should be `fetchOrderBook (symbol, limit = undefined, params = {})`